### PR TITLE
terminal component: make the prompt check less fragile

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8332/

File: jujugui/static/gui/src/app/components/terminal/terminal.js [hunk 1]
```diff
@@ -99,7 +99,8 @@ class Terminal extends React.Component {
           if (this.terminalSetup && !this.initialCommandsSent) {
             // If the first PS1 presented to the user changes then this will
             // need to be updated.
-            if (resp[1].indexOf('\u001b[01;32') === 0) {
+            const suffix = '$ ';
+            if (resp[1].indexOf(suffix, resp[1].length-suffix.length) !== -1) {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/terminal/terminal.js]: Could be `if (resp[1].slice(-suffix.length) === suffix) {` which might be a little more readable, but probably just a preference thing :).

**Comment by frankban** [comment]
Thanks for the review!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**